modules/v2x/BUILD

```diff
@@ -1,4 +1,3 @@
-load("@rules_cc//cc:defs.bzl", "cc_binary")
 load("//tools:cpplint.bzl", "cpplint")
 
 package(default_visibility = ["//visibility:public"])
```

Comment:
When was this `v2x_proxy_lib` merged into Apollo codebase since it contains a `main` function?

Reply:
maybe 2 years ago